src/Illuminate/Support/Stringable.php

```diff
@@ -396,7 +397,6 @@ public function lower()
     /**
      * Convert GitHub flavored Markdown into HTML.
      *
-     * @param  array  $options
```

Comment:
Why this was removed? (and 4 other `@param` and one `@return` at `jsonSerialize`)

Reply:
Hum weird indeed, I wrote that directly in github.dev, the ide should have removed them.